bin/pgenv

```diff
@@ -4,11 +4,53 @@
 trap 'exit' ERR
 set -E
 
+# message levels, used by pgenv_message
+declare -A msg_levels
+msg_levels[debug]=5
+msg_levels[info]=10
+msg_levels[warn]=20
+msg_levels[error]=25
+msg_levels[always]=999
```

Comment:
These levels are decreasingly verbose as the numbers go up, until `always`, which is the most verbose. Shouldn't it be 1 (or 0)? Or should the levels be reversed other than `always`?

Reply:
Uhm...not seeing commits 390a13916d926cbe4f7ce27590f000637713a005 and df56220e7e83ba35ad6993d72a3b285fb6baa99b from my branch here.
The two commits above should solve your questions.

Reply:
Uhm...not seeing commits 390a13916d926cbe4f7ce27590f000637713a005 and df56220e7e83ba35ad6993d72a3b285fb6baa99b from my branch here.
The two commits above should solve your questions.